quickstart/.env

```diff
@@ -11,8 +11,8 @@ DOCKER_NETWORK=quickstart
 # canton
```

Comment:
probably not for this PR but this file seems to be getting out of hand. You probably wanna at a minimum split it, e.g., into env for app provider, end user, … and I'd also consider using some sort of yaml config instead where you can have a decent hierarchy